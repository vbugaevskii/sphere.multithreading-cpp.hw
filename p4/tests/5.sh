sleep 0.1 && false || echo running in background | cat &
echo pre
sleep 0.2
echo post

echo exit 71 |  bash &
sleep 0.1
