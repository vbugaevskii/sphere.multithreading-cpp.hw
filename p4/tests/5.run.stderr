Spawned child process 13964
Process 13966 exited: 0
Process 13965 exited: 0
Process 13968 exited: 1
Process 13969 exited: 0
Process 13967 exited: 0
Process 13971 exited: 0
Spawned child process 13972
Process 13974 exited: 0
Process 13975 exited: 71
Process 13972 exited: 0
Process 13973 exited: 0
Process 13970 exited: 0
